rust/exchange_rate/vite.config.js

```diff
@@ -0,0 +1,31 @@
+const isProduction = process.env.DFX_NETWORK === "ic";
+
+import { svelte } from "@sveltejs/vite-plugin-svelte";
+import path from "path";
+import { defineConfig } from "vite";
+import EnvironmentPlugin from "vite-plugin-environment";
+
+export default defineConfig({
+  mode: isProduction ? "production" : "development",
+  root: path.resolve(__dirname, "src", "frontend"),
+  build: {
+    outDir: path.resolve(__dirname, "src", "frontend", "public"),
+    emptyOutDir: true,
+  },
+  define: {
+    global: "window",
```

Comment:
have you try a prod build? nothing else to polyfill?

`npm run build && npx serve dist`